.github/workflows/rust.yml

```diff
@@ -84,6 +94,17 @@ jobs:
       - name: Build Release
         run: cargo build --verbose --release $RUST_TARGET_FLAG
 
+      # Cross compilation
+
+      - name: Build aarch64 macOS
+        if: ${{ matrix.os == 'macos-latest' }}
+        run: cargo build --verbose --target aarch64-apple-darwin
+
+      # TODO: #42 linking fails here for some reason
```

Comment:
Should we add an explicit note to issue 42 saying linking fails when cross compiling on aarch64 ubuntu?

Reply:
Done.